internal/dumpfile/parse.go

```diff
@@ -0,0 +1,138 @@
+// Package dumpfile implements encoding and decoding of Edwood dump file.
+//
+// A dump file stores the state of Edwood so that it can be restored
+// when Edwood is restarted.
+package dumpfile
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+)
+
+const version = 1
+
+// WindowType defines the type of window.
+type WindowType int
+
+const (
+	Saved   WindowType = iota // Saved is a File and directory stored on disk
+	Unsaved                   // Unsaved contains buffer that's not stored on disk
+	Zerox                     // Zerox is a copy of a Saved or Unsaved window
+	Exec                      // Exec is a window controlled by an outside process
+)
+
+// Content stores the state of Edwood.
+type Content struct {
+	CurrentDir string
+	VarFont    string // Variable width font
+	FixedFont  string // Fixed width font
+	RowTag     string
+	Columns    []Column
+	Windows    []Window
+}
+
+// Column stores the state of a column in Edwood.
+type Column struct {
+	Position float64 // Position within the row (in percentage)
+	Tag      string
+}
+
+// Window stores the state of a window in Edwood.
+type Window struct {
+	Type WindowType
+
+	ColumnID int
```

Comment:
This is the Column index where the window will be placed, which I think should be preserved. I renamed the field to `Column`, so it may be less confusing.